pontoon/base/templates/translate.html

```diff
@@ -218,101 +238,155 @@ <h3 class="loading"><div class="fa fa-refresh fa-spin"></div></h3>
-        <button id="save" title="Save Translation (Enter)"></button>
-      {% else %}
-        <p id="sign-in-required"><a id="sidebar-signin" href="">Sign in</a> to translate.</p>
-      {% endif %}
-    </menu>
-
-    <div id="helpers" class="tabs">
-      <nav>
+      <nav id="plural-tabs">
         <ul>
-          <li class="active">
-            <a href="#history">
-              <span>History
-                <span class="fa fa-cog fa-lg fa-spin"></span>
-                <span class="count"></span>
-              </span>
-            </a>
-          </li><li>
-            <a href="#machinery">
-              <span>Machinery
-                <span class="fa fa-cog fa-lg fa-spin"></span>
-                <span class="count"></span>
-              </span>
-            </a>
-          </li><li>
-            <a href="#other-locales">
-              <span>Locales
-                <span class="fa fa-cog fa-lg fa-spin"></span>
-                <span class="count"></span>
-              </span>
-            </a>
-          </li>
+          <li class="active"><a href="#plural1"><span></span><sup></sup></a></li>
+          <li><a href="#plural2"><span></span><sup></sup></a></li>
+          <li><a href="#plural3"><span></span><sup></sup></a></li>
+          <li><a href="#plural4"><span></span><sup></sup></a></li>
+          <li><a href="#plural5"><span></span><sup></sup></a></li>
+          <li><a href="#plural6"><span></span><sup></sup></a></li>
         </ul>
       </nav>
 
-      <section class="history">
-        <ul></ul>
-      </section>
+      <textarea {% if not user.is_authenticated() %} readonly {% endif %} dir="auto" lang="{{ locale.code }}" id="translation" placeholder="Type translation and press Enter to save"></textarea>
+
+      <menu>
+        {% if user.is_authenticated() %}
+          <div id="unsaved" class="warning-overlay">
+            <a href="#" class="cancel">&times;</a>
+            <h2>You have unsaved changes</h2>
+            <p>Sure you want to leave?</p>
+            <button id="leave-anyway">Leave anyway</button>
+          </div>
+          <div id="quality" class="warning-overlay">
+            <a href="#" class="cancel">&times;</a>
+            <h2>The following checks have failed</h2>
+            <ul></ul>
+            <button id="save-anyway"></button>
+          </div>
+          <div id="translation-length">
+            <span class="current-length"></span>|<span class="original-length"></span>
+          </div>
+          <button id="copy" title="Copy From Source (Ctrl + Alt + C)">Copy</button>
+          <button id="clear" title="Clear Translation (Ctrl + Alt + Backspace)">Clear</button>
+          <button id="save" title="Save Translation (Enter)"></button>
+        {% else %}
+          <p id="sign-in-required"><a id="sidebar-signin" href="">Sign in</a> to translate.</p>
+        {% endif %}
+      </menu>
+
+      <div id="helpers" class="tabs">
```

Comment:
i guess we could refactor these tabs into jinja macros. However, maybe its a thing for the separate task.